iModelCore/ECDb/ECDb/BuiltInFuncs.cpp

```diff
@@ -113,6 +113,7 @@ void ExtractPropFunc::_ComputeScalar(Context& ctx, int nArgs, DbValue* args) {
 
         InstanceReader::JsonParams params;
         params.SetUseJsName(jsonFlags & InstanceReader::FLAGS_UseJsPropertyNames);
+        params.SetClassIdToClassNames(jsonFlags & InstanceReader::FLAGS_UseJsPropertyNames);
```

Comment:
You have been setting SetClassIdToClassNames on UseJsPropertyNames flag being set - that could explain why you removed quite a few USE_JS_PROP_NAMES in ECSqlRowAdaptor

Reply:
The setting of the `SetClassIdToClassNameson` on `UseJsPropertyNames` flag is intentional.
We only support the `FLAGS_DoNotTruncateBlobs` and `FLAGS_UseJsPropertyNames` flags. The `FLAGS_SetClassIdToClassNames` flag is not supported. The value set by the FLAGS_UseJsPropertyNames flag was previously used in `iModelCore\ECDb\ECDb\ECSqlRowAdaptor.cpp` on [Line 175](https://github.com/iTwin/imodel-native/blob/aks/ecsql-statement-js-binding/iModelCore/ECDb/ECDb/ECSqlRowAdaptor.cpp#L175) and [Line 290](https://github.com/iTwin/imodel-native/blob/aks/ecsql-statement-js-binding/iModelCore/ECDb/ECDb/ECSqlRowAdaptor.cpp#L290)